docs/pt_BR/building_your_app.md

```diff
@@ -0,0 +1,11 @@
+# Construindo seu App
+
+electron-vue dá suporte aos dois [electron-packager](https://github.com/electron-userland/electron-packager) e [electron-builder](https://github.com/electron-userland/electron-builder) para construir e distruibir sua aplicação pronta para produção. As duas ferrametas para buildar são apoiadas pela incrível comunidade [@electron-userland](https://github.com/electron-userland) e tem uma documentação detalhada. Durante `vue-cli` a estruturação você será perguntado quanto ao builder que deseja usar.

```

Comment:
- Change `ferrametas` to `ferramentas`
- Change `Durante vue-cli a estruturação você será perguntado quanto ao builder que deseja usar.` to `Durante a instalação do vue-cli, você será questionado quanto ao builder que deseja usar.`